src/util.js

```diff
@@ -0,0 +1,19 @@
+const { createError } = require("micro");
+
+const addSizeToUrl = function(url, size) {
+  const exists = url.indexOf("SX300");
+
+  // doesn't exist
+  if (exists === -1) {
+    throw createError(
+      400,
+      `Unable to fetch movie poster with specified size ${size}`
```

Comment:
This is not really failing because the size in `{size}` does not exist, but because the input url is not valid at all (i.e., does not contain any size)? I would change the error message to "Unable to fetch .. because the input url is invalid" or similar).